source/connect.txt

```diff
@@ -105,6 +105,46 @@ deployment from which you were disconnected:
 
 .. include:: /includes/steps/activate-connection.rst
 
+Launch MongoDB Shell
+~~~~~~~~~~~~~~~~~~~~
+
+You can connect the
+`MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
+:binary:`mongo <mongo>` shell to your current active deployment.
+
+Considerations
+``````````````
+
+- |vsce| determines which shell to use to connect to your deployment
+  by the :guilabel:`Shell` setting in your
+  :ref:`extension settings <vsce-settings>`. You can choose either the
+  `MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
```

Comment:
[nit] I feel like the first sentence could be simplified, maybe -

The shell |vsce| uses to connect to your deployment is determined by the :guilabel:`Shell` setting in your :ref:`extension settings <vsce-settings>`.

Reply:
That was my original wording but I modded it to the current one for a bit more active voice. I think ultimately you're right that your suggestion reads more cleanly. Updating.